engines/grim/emi/modelemi.cpp

```diff
@@ -36,10 +36,10 @@
 namespace Grim {
 
 struct Vector3int {
-	int _x;
-	int _y;
-	int _z;
-	void setVal(int x, int y, int z) {
+	uint16 _x;
+	uint16 _y;
+	uint16 _z;
+	void setVal(uint16 x, uint16 y, uint16 z) {
```

Comment:
Won't this break the PS2 models?

Reply:
I've just tried the PS2 version and it still works.
At least the introduction, the boat chapter and the beginning on Melee Island.